gvr-javascript/assets/script.js

```diff
@@ -1,29 +1,41 @@
+importPackage(org.gearvrf)
+importPackage(org.gearvrf.scene_objects)
+
 function onInit(gvrf) {
   var mainScene = gvrf.getNextMainScene();
 
-  // 3D model
-  var model = gvrf.loadModel("astro_boy.dae");
-  model.getTransform().setRotationByAxis(45.0, 0.0, 1.0, 0.0);
-  model.getTransform().setScale(3, 3, 3);
-  model.getTransform().setPosition(0.0, -0.4, -0.5);
-
-  mainScene.addSceneObject(model);
+  // 3D Boat Scene
+  var boat = gvrf.loadModel("RowBoatFBX/RowBoatAnimated.fbx");
+  boat.setName("boat");
+  boat.getTransform().setScale(.5, .5, .5);
+  boat.getTransform().setRotationByAxis(20, 0, 1, 0);
+  boat.getTransform().setPosition(20, -20, -40);
+  mainScene.addSceneObject(boat);
 
   // Text
-  var textView = utils.newTextViewSceneObject(gvrf, "GVRf scripting in Javascript");
+  var textView = new GVRTextViewSceneObject(gvrf, gvrf.getActivity());
+  textView.setText("GVRf scripting in Javascript");
+  textView.setRefreshFrequency(GVRTextViewSceneObject.IntervalFrequency.HIGH);
   var textSize = textView.getTextSize();
+  textView.setName("text");
   textView.setTextSize(textSize);
```

Comment:
Please set the text size to something in the 6-8sp range. Looks decent. textView.getTextSize() returns number of pixels but textView.setTextSize() works with scaled pixels and thus setTextSize(getTextSize()) blows up the text. Also please use var textView = new GVRTextViewSceneObject(gvrf) instead of var textView = new GVRTextViewSceneObject(gvrf, gvrf.getActivity()).